rtlib/include/rt/ImageStitcher.hpp

```diff
@@ -0,0 +1,93 @@
+#pragma once
+
+#include <vector>
+#include <iostream>
+
+#include <opencv2/stitching.hpp>
+
+namespace rt
+{
+
+class ImageStitcher
+{
+public:
+    ImageStitcher() = default;
+
+    // Sets the images to be stitched
+    void setImages(cv::InputArrayOfArrays i) { i.getUMatVector(imgs_); }
+
+    // Stitches the images together
+    cv::Mat compute();
+
+    // Sets the user generated landmarks
+    // This will probably need to change so that there can be more than 2 images
+    void setLandmarks(std::vector<std::string> ldmFiles);
+
+    // Sets the bool to whether or not landmarks should be generated
+    void setGenerateLandmarks(bool generate){ generate_landmarks_ = generate; }
+
+    void setOption(int option){ option_ = option; }
+
+    void printFeatures(std::string file_path);
+
+    void printMatches(std::string file_path);
```

Comment:
My thought behind adding these was for debugging purposes so that if a stitch looked weird the user could see the automatically generated features and matches. Is that something that we'd want to keep in there?